Backend/UserService.cs

```diff
@@ -132,7 +127,7 @@ public async Task<IdentityResult> UpdateUserAsync(User userToUpdate)
 
         public UserInformations GetUserFromLDAP(string login, string password, ILogger logger)
         {
-            login += _ldapSettings.MailPrefix;
+            login += "@fri.uniza.sk";
```

Comment:
This pull request is for reverting university ldap merge. i wan't to revert it because it is not working and we want to test new features and fixes on staging. This should put our develop branche into state before merging branch with university ldap feature.